requirements.txt

```diff
@@ -1,21 +1,21 @@
-libpysal>=4.5.1
+libpysal>=4.5.1.post2
 access>=1.1.3
 esda>=2.4.1
 giddy>=2.3.3
 inequality>=1.0.0
 pointpats>=2.2.0
-segregation>=2.0.0
-spaghetti>=1.6.2
+segregation>=2.1.0
+spaghetti>=1.6.4
 mgwr>=2.1.2
-momepy>=0.5.0
+momepy>=0.5.1
```

Comment:
`momepy` should be [`0.5.2`](https://github.com/pysal/momepy/releases/tag/v0.5.2)

Reply:
see above for spaghetti